tests/integration/checkpoint.bats

```diff
@@ -441,3 +441,38 @@ function simple_cr() {
 	pid=$(cat "pid")
 	grep -q "${REL_CGROUPS_PATH}$" "/proc/$pid/cgroup"
 }
+
+@test "checkpoint/restore and exec" {
+	runc run -d --console-socket "$CONSOLE_SOCKET" test_busybox
+	[ "$status" -eq 0 ]
+
+	testcontainer test_busybox running
+
+	local execed_pid=""
+	for _ in $(seq 2); do
+		# checkpoint the running container
+		runc checkpoint --work-path ./work-dir test_busybox
```

Comment:
I don't remember, is the dump cleaned-up after the test runs?

Reply:
Everything that is created under `$PWD` is cleaned up (look for `$ROOT` in `setup_bundle` and `cleanup_bundle`).

Reply:
Thanks!